cmd/tracee-ebpf/main.go

```diff
@@ -83,12 +83,8 @@ func main() {
 			cfg.Output = &output
 
 			// kernel lockdown check
-
 			lockdown, err := helpers.Lockdown()
-			if err != nil {
-				return err
-			}
-			if lockdown == helpers.CONFIDENTIALITY {
+			if err == nil && lockdown == helpers.CONFIDENTIALITY {
```

Comment:
It sounds like an error can happen in two cases:
1. File was not found
2. File was found but had irrelevant content

In both cases it seems we want to proceed ahead. If so, does it make sense to return an error from this function?

Reply:
That was discussed at https://github.com/aquasecurity/libbpfgo/pull/103#discussion_r780512300 and returning the error seemed appropriate. Also, we might have other users of libbpfgo that might want to know if the file didn't exist or the error was on its contents (for example). Makes sense ?

Reply:
You're right. The other users part convinced me to keep it. Thanks.